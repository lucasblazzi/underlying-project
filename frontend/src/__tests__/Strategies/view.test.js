import React from 'react';
import ViewStrategy from '../../pages/Strategies/view';
import renderer from 'react-test-renderer';

it('PÃ¡gina', () => {
    const tree = renderer.create(<div class = "page-content"/>).toJSON();
    expect(tree).toMatchSnapshot();
})

it('CardBody', () => {
    const tree = renderer.create(<div class="m-3" />).toJSON();
    expect(tree).toMatchSnapshot();
})