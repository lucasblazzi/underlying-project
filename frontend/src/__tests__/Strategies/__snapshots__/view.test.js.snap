// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody 1`] = `
<div
  class="m-3"
/>
`;

exports[`PÃ¡gina 1`] = `
<div
  class="page-content"
/>
`;
